casepro/cases/models.py

```diff
@@ -103,6 +103,16 @@ def __str__(self):
         return self.name
 
 
+class SystemUser(User):
+
+    @classmethod
+    def get_or_create(cls):
+        systemUsers = cls.objects.filter()
+        if len(systemUsers) > 0:
+            return systemUsers[0]
```

Comment:
I think django will create a separate `SystemUser` table here (with hopefully only one entry), but I could be wrong.